sunpy/util/util.py

```diff
@@ -212,3 +212,37 @@ def expand_list_generator(inp):
                 yield nested_item
         else:
             yield item
+
+def partial_match(key, dictionary):
+    """
+    Return value/values based on the partial key. Each element of the partial key is matched against the keys of the dictionary and if a match is found 
+    the value of the key is returned.
+    Even a partial match works here i.e even if the key matches partially
+    a value is returned. Note that this function uses a generator so we need to type cast the output to a list to get the values.
+
```

Comment:
The doc string here. You want to have a new line at a full stop and re-flow the text after if its too long.

Reply:
Um sorry I didn't get that. You mean I should have a new line after every full stop?

Reply:
Yes.